src/identity/verification.ts

```diff
@@ -1050,6 +458,7 @@ export function* tryRevealPhoneNumber(
       smsRetrieverAppSig,
       language,
       securityCodePrefix: getSecurityCodePrefix(issuer),
+      phoneNumberSignature: undefined,
```

Comment:
you might also be able to use `phoneHashDetails.unblindedSignature` here... but it looks like unused code anyway...